int32 num1
int32 num2
int32 clientID
---
int32 result